queries/hyprlang/highlights.scm

```diff
@@ -0,0 +1,54 @@
+(comment) @comment
+
+(source "source" @keyword)
+
+[ "exec" "exec-once" ] @keyword
```

Comment:
probably `@keyword` is correct for `source`, `exec`, `exec-one`, `bind`  which probably fits nicely with the highlighting as then every line starts with a keyword. I was wondering whether they could also be something `function.builtin` like has they have something of a command. I have currently no opinion which better fits the semantics.

Reply:
Oh, I forgot to add source with the other ones. 

I just used the keyword group because they're called [keywords](https://wiki.hyprland.org/Configuring/Keywords/) but I also think function makes more sense in a way but it could also be confusing because it's just a key value relation but not a real programming language.